packages/rust/db/migrations/20220528030155_initial.up.sql

```diff
@@ -1,93 +1,69 @@
 CREATE TABLE "user"
 (
-    id              serial PRIMARY KEY,
+    id              SERIAL PRIMARY KEY,
     uuid            UUID UNIQUE         NOT NULL,
     username        VARCHAR(255) UNIQUE NOT NULL,
-    password        VARCHAR(255)            NULL,
-    profile_picture VARCHAR(255) NULL,
+    password        VARCHAR(255)        NOT NULL,
     email           VARCHAR(255) UNIQUE NULL,
-    phone           VARCHAR(11) UNIQUE NULL,
```

Comment:
Removing phone #?

Reply:
I don't think we actually need it. Maybe later we can add back?

Reply:
Not a necessary thing to store right now, but I think it is definitely something we want in the final product. Allows another way to verify a person and would prevent bots if it was required. Unsure if we would make it required though.